### PR TITLE
[doc] Improve operator page

### PR DESCRIPTION
**Pull request by qiao-bo**

### CODE REVIEW

**Review by Vissidarte-Herman — APPROVED** [review]
LGTM

File: docs/lang/articles/basic/operator.md [hunk 1]
```diff
@@ -83,6 +83,15 @@ default numerical types.
 | `a << b`                | left-shift `a` by `b` bits          |
 | `a >> b`                | right-shift `a` by `b` bits         |
 
+:::note
+
+The `>>` operation denotes the
+[Shift Logical](https://en.wikipedia.org/wiki/Logical_shift) Right (SHR) operation. For the
```
Comment (lin-hitonami) [inline docs/lang/articles/basic/operator.md:89]: `>>` is sar

File: docs/lang/articles/basic/operator.md [hunk 2]
```diff
@@ -83,6 +83,15 @@ default numerical types.
...
+[Shift Arithmetic](https://en.wikipedia.org/wiki/Arithmetic_shift) Right (SAR) operation,
+consider using `ti.bit_shr()` or `ti.bit_shl()` for the left shift operation.
```
Comment (lin-hitonami) [inline docs/lang/articles/basic/operator.md:91]: sal and shl are synonyms so we can just use `<<`

File: docs/lang/articles/basic/operator.md (hunk 1 above)
Comment (qiao-bo) [reply docs/lang/articles/basic/operator.md]: right, messed up, fixed

File: docs/lang/articles/basic/operator.md (hunk 2 above)
Comment (qiao-bo) [reply docs/lang/articles/basic/operator.md]: ok, let's mention both for completeness.

**Review by lin-hitonami — APPROVED** [review]
LGTM!

**Pull request merged**